spring-integration-zmq/src/main/java/org/springframework/integration/zmq/config/xml/ZmqMessageDrivenChannelAdapterParser.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2002-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.zmq.config.xml;
+
+import org.w3c.dom.Element;
+
+import org.springframework.beans.factory.support.AbstractBeanDefinition;
+import org.springframework.beans.factory.support.BeanDefinitionBuilder;
+import org.springframework.beans.factory.xml.ParserContext;
+import org.springframework.integration.config.xml.AbstractChannelAdapterParser;
+import org.springframework.integration.config.xml.IntegrationNamespaceUtils;
+import org.springframework.integration.zmq.inbound.ZmqMessageDrivenChannelAdapter;
+
+/**
+ * The ZmqAdapter Message Driven Channel adapter parser
+ *
+ * @author Subhobrata Dey
+ * @since 5.1
+ *
+ */
+public class ZmqMessageDrivenChannelAdapterParser extends AbstractChannelAdapterParser {
```

Comment:
As well as the naming concern for classes. 
I'm pretty fine to keep them as `Zeromq...`